client/app/bundles/momentCard/components/MomentCardDate.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import css from './MomentCard.scss';
+
+type MomentCardDateState = {};
+
+type MomentCardDateProp = {
+  date: string;
+};
+
+export default class MomentCardDate
+  extends React.Component<MomentCardDateProp, MomentCardDateState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.